pkg/asset/installconfig/ovirt/validators.go

```diff
@@ -37,3 +40,32 @@ func authenticated(c *Config) survey.Validator {
 	}
 
 }
+
+//Validate executes ovirt specific validation
+func Validate(c *types.InstallConfig) error {
+	con, err := NewConnection()
+	if err != nil {
+		return err
+	}
+	defer con.Close()
+
+	return validateVNICProfile(*c.Ovirt, con)
+}
+
+func validateVNICProfile(p ovirt.Platform, con *ovirtsdk.Connection) error {
+	if p.VNICProfileID == "" {
```

Comment:
This validation is providing a default value, but I think you are already doing that elsewhere. Validation should be checking that if a user provides a value then it is valid.

Reply:
The way I am reading this code:
if user provided a vnic profile ID: do nothing (it is valid)
if user did not provide a vnic profile ID: see if you can determine a default or else return an error

What I was expecting:
if user provided a vnic profile ID: validate (use API to determine if user provided value is correct)
if user did not provide: do nothing (default will be assigned later)

Does that make sense? I could definitely have mistaken assumptions here.

Reply:
ack